server/src/controllers/orderController.ts

```diff
@@ -32,9 +32,21 @@ orderController.post(
               object.payment_intent,
               object.id,
           );
-        // handle donation amount in metadata (will be a donation team task?)
         } else if (metaData.sessionType === '__donation') {
-        // donation is handled
+          // Create a date model for the date field in the donations table
+          await prisma.donations.create({
+            data: {
+              contactid_fk: Number(metaData.contactID),
+              isanonymous: metaData.anonymous,
+              amount: metaData.donation,
+              donorname: metaData.firstName + ' ' + metaData.lastName,
+              frequency: 'one_time',
+              comments: metaData.comments,
+              payment_intent: object.payment_intent,
+              refund_intent: object.refund_intent,
+              donationdate: Number(new Date().toISOString().slice(0, 10).replace(/-/g, '')),
+            },
+          });
```

Comment:
It might be worth checking the checkout session status before recording the transaction, just in case it failed.

Reply:
I will get that added in this weekend and thank you for reviewing this MR for me.

Reply:
Another thing to take a look at would be leaving the refund intent null unless the donation has been refunded.